modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -743,16 +757,18 @@ private void processOperation(ChannelHandlerContext ctx, ClientMessageUnpacker i
                 return ClientJdbcColumnMetadataRequest.process(in, out, jdbcQueryEventHandler);
 
             case ClientOp.JDBC_SCHEMAS_META:
-                return ClientJdbcSchemasMetadataRequest.process(in, out, jdbcQueryEventHandler);
+                return ClientJdbcSchemasMetadataRequest.process(in, out, jdbcQueryEventHandler).thenRun(() -> {
+                    out.meta(clockService.current());
```

Comment:
why we need to update timestamp when requesting metadata?
why most time we updating it conditionally?
```
if (!(out.meta() instanceof HybridTimestamp))
```

Reply:
We ought to have a timestamp more or equal to the timestamp that is used to get the metadata. Otherwise, we may start RO before a table in the metadata was created.

Reply:
`if (!(out.meta() instanceof HybridTimestamp))`
This condition is useless for the metadata request because we never calculate an observation timestamp for this request type.